sdks/java/maven-archetypes/examples/build.gradle

```diff
@@ -28,6 +28,7 @@ processResources {
     'project.version':  version,
     'bigquery.version': dependencies.create(project.library.java.google_api_services_bigquery).getVersion(),
     'google-clients.version': dependencies.create(project.library.java.google_api_client).getVersion(),
```

Comment:
Maybe we should rename `google-clients.version` to better distinguish from `google-http-client`?

Reply:
That's a nice idea.

Reply:
Done.